lib/known_bugs.pm

```diff
@@ -28,12 +28,30 @@ our @EXPORT = qw(
   upload_journal
 );
 
+
+=head1 KNOWN_BUGS
+
+=head1 SYNOPSIS
+
+C<use lib/known_bugs>
+
+Allows detection of known errors on the serial console
+
+As we have reocurring problems that can be easily detected on serial level we have decided to detect and show them in opneQA to ease up review and where possible only softfail to not lose the whole test suite
+
+=cut
+
+=head2 create_list_of_serial_failures
```

Comment:
I am missing the synopsis of the functions. @foursixnine ?

https://github.com/os-autoinst/os-autoinst/blob/f48d4fc7579845f802c0010ec3ca6a9607d9d844/testapi.pm#L135

```suggestion
=head2 create_list_of_serial_failures

    $ref_to_list = create_list_of_serial_failures()
```

Reply:
There is no synopsis, as this function is not intended to be called by the end-user, but only to be extended with new serial failures

Reply:
I thought this documentation is for the code in general, not only for end-users.

Reply:
I agree with @SergioAtSUSE's comment. However I'd use it differently, mostly because we should ilustrate the test developer how to interact with this. While it is documented at the os-autoinst/distribution.pm level, extending a bit here, doesn't hurt.

```suggestion
=head2 create_list_of_serial_failures
    my $list = create_list_of_serial_failures();
    $testapi::distri->set_expected_serial_failures($list);
    
```

Reply:
but, don't forget the space after `=head2`